### ISSUE TITLE
`Body Flares` are being deactivated when `Dynamic Dimming Skies` are

### ISSUE THREAD

**Issue by Lisias** [net-lisias-ksp/DistantObject#32]
Fellow Kerbonaut [Why485](https://forum.kerbalspaceprogram.com/index.php?/profile/26795-why485/) [reported](https://forum.kerbalspaceprogram.com/index.php?/topic/205063-ksp-131-distant-object-enhancement-doe-l-under-new-management-21111r2-2023-0116/&do=findComment&comment=4252278) this unexpected bug:

> This is one of those mods that I've been using since forever, but I don't recall having as much of an issue with the dimming as I am now. Is it intentional that when "Dynamic Sky Dimming" is disabled, all the planet flares no longer render? Basically if you disable the dimming, the mod seems to do nothing.

![Dsxlpec](https://user-images.githubusercontent.com/64334/222946623-8e38095a-5c01-4c4f-be15-df44598fae6b.gif)

Perhaps I had messed on choosing the layer for the flares while trying to work out them being visible on Atmospheres?

**Comment by Lisias** [comment]
Since I could not reproduce it on a vanilla KSP, it's surely related to a interaction with a 3rd party code. So this is not a bug anymore, and become a support issue.

**Comment by Lisias** [comment]
Fellow Kerbonaut [Kurgut](https://forum.kerbalspaceprogram.com/profile/188918-kurgut/) reported another [occurrence](https://forum.kerbalspaceprogram.com/topic/205063-ksp-131-distant-object-enhancement-doe-l-under-new-management-21114-2023-0818/?do=findComment&comment=4355340) from this problem.

With logs! :)

[Uploading KSP.log…]()

Let's see if I can zero in the problem now!!

**Comment by Lisias** [comment]
Sooo…. Fellow Kerbonaut [reported](https://forum.kerbalspaceprogram.com/topic/205063-ksp-131-distant-object-enhancement-doe-l-under-new-management-21114-2023-0818/?do=findComment&comment=4355400) that they could reproduce the problem on a vanilla KSP installment!!!! :-/

New logs:
* [KSP.log](https://github.com/net-lisias-ksp/DistantObject/files/13823542/KSP.log)
* [Player.log](https://github.com/net-lisias-ksp/DistantObject/files/13823543/Player.log)

Now I have two problems to cope with - this one, obviously, and why in hell I failed to reproduce this issue last time I tried. (sigh)

**Comment by Lisias** [comment]
I'm a fucking idiot. If I remember correctly, I didn't did a full cycle - I had checked if the `SettingsGUI` were doing its job as expected, as I had made a mistake like that in https://github.com/net-lisias-ksp/DistantObject/issues/38 and  https://github.com/net-lisias-ksp/DistantObject/issues/14 .

Not to mention https://github.com/net-lisias-ksp/DistantObject/commit/189669f199d057c979de615da5cb4f361d7d1459 . 

Problem - this time I screwed on the FlareDraw class itself. I didn't reproduced the problem last time I tried because I wasn't testing the problem, but what I was thinking it was the cause of the problem directly.

Bad mistake.

Well, finally fixed in https://github.com/net-lisias-ksp/DistantObject/commit/baa8113c4b74c73c9537b560e888e892b2c32ad5 . Now I will reproduce the problem properly, and then I will verify the fix. No more fucking testing mishaps this time - at least, not the ones I'm aware of...
